subt/main.py

```diff
@@ -260,9 +260,9 @@ def go_safely(self, desired_direction):
         desired_angular_speed = 0.9 * safe_direction
         size = len(self.scan)
         dist = min_dist(self.scan[size//3:2*size//3])
+        safety_factor = 2.0
         if dist < self.min_safe_dist:  # 2.0:
-#            desired_speed = self.max_speed * (1.2/2.0) * (dist - 0.4) / 1.6
-            desired_speed = self.max_speed * (dist - self.dangerous_dist) / (self.min_safe_dist - self.dangerous_dist)
```

Comment:
well, this will break/make unusable extra parameters for Kloubak K2/K3. In general I agree with this idea, just it should be propagated there too (to JSON configs)

Reply:
@tajgr - please check this on K2 and remove obsolete params

Reply:
done, it is better so we are merging